rest-api/core-rest/src/main/java/uk/gov/gchq/gaffer/rest/service/IOperationService.java

```diff
@@ -51,6 +52,11 @@
     Object execute(final OperationChain opChain);
 
     @POST
+    @Path("/operation")
```

Comment:
I know there are a lot of changes in version 0.7.0 already, but that could be quite an annoying breaking change. It makes this tidier but doesn't really give any benefit. Perhaps something we think about doing at a later date.